spark_pipeline_framework/utilities/fhir_helpers/fhir_sender_helpers.py

```diff
@@ -47,6 +48,11 @@ def send_json_bundle_to_fhir(
 
     if retry_count is not None:
         fhir_client = fhir_client.retry_count(retry_count)
+    if additional_request_headers is not None:
+        logger.debug(
```

Comment:
logger.debug will not work here.
reason: getlogger is used and its default level is info. so this will only work if we set the level to debug. here you should have used info. change everywhere